packages/Autodiscovery/src/Rector/FileSystem/MoveServicesBySuffixToDirectoryRector.php

```diff
@@ -92,7 +92,7 @@ private function processGroupNamesBySuffix(
             }
 
             // is suffix in the same category, e.g. "Exception/SomeException.php"
-            $expectedLocationFilePattern = sprintf('#\/%s\/.+%s#', preg_quote($groupName), preg_quote($suffixPattern));
+            $expectedLocationFilePattern = sprintf('#\/%s\/.+%s#', preg_quote($groupName, '#'), preg_quote($suffixPattern, '#'));
```

Comment:
@TomasVotruba Maybe the usage of "preg_quote" need to be checked in "ConsistentPregDelimiterRector"?

Reply:
Indeed. Care for PR to add it?

Reply:
It seems that "ConsistentPregDelimiterRector" will only work for simple strings without variables etc. So "preg_quote" is not effected, because it will not be replaced anyway, correct?

---

PS: I tried to debug some stuff but I don't know why I can't simply "var_dump" something in e.g. "ConsistentPregDelimiterRector"? My laptop will crash while running the phpunit tests, it also crash if I add a "exit(1)" after the "var_dump"? Is there a debug option or something like that?

e.g.:
```php
        if (! $arg->value instanceof String_) {
var_dump($arg); exit();
            return;
        }
```